sureal/pc_subjective_model.py

```diff
@@ -265,11 +265,12 @@ def resolve_model(alpha, **more):
         vari_p = np.diagonal(cova_p)[:-1]
         stdv_p = np.sqrt(vari_p)
         cova_p = cova_p[:-1, :-1]
+        cova_v = cova_p / (p**2)
```

Comment:
Is this correct? `cova_p` is a square matrix and `p**2` is a vector. What does this operation do? And what is its physical meaning?

Reply:
If you go to your overleaf document https://fr.overleaf.com/project/5f088492262bcc000111f9b3 Pair Comparison (V0), end of part 2. As p_i is defined as being exp(\psi_i), we conclude that var(\psi_i) is var(pi)/pi^2. So the issue here is that we consider the variance of p instead of the variance of \psi

You are right, the right mathematical matricial notations is cova_v = cova_p / (p*p.T) and then cova_v = cova_p / (np.expand_dims(p, axis=1) * (np.expand_dims(p, axis=1).T)). A simple verification I have done now is that we should have stdv_v == np.sqrt(np.diag(cova_v))

Reply:
Should I change it and update the PR?

Reply:
Yes please update. also, could you output both cova_v and cova_p? Keep both `quality_scores_p_cov` and `quality_scores_v_cov` in the dictionary.

`cova_v = cova_p / (p*p.T)` => i think you can also do: `cova_p / np.array((np.matrix(p).T * np.matrix(p)))`?? Please double check